src/hera/workflows/v5/examples/upstream/dag_diamond.py

```diff
@@ -0,0 +1,32 @@
+from hera.workflows.models import Arguments, Parameter
+from hera.workflows.v5.container import Container
+from hera.workflows.v5.dag import DAG
+from hera.workflows.v5.task import Task
+from hera.workflows.v5.workflow import Workflow
+
+# Showing the flexibility of hera v5
+# here we defined the echo container ealier
+# but add it to the workflow towards the end
+# to ensure we can exactly match the upstream argo
+# example and have the ability to control the output
+# workflow, down to the level of being able to control the
+# output list of templates
+echo = Container(
+    name="echo",
+    image="alpine:3.7",
+    command=["echo", "{{inputs.parameters.message}}"],
+    inputs=[Parameter(name="message")],
+)
+with Workflow(
+    generate_name="dag-diamond-",
+    entrypoint="diamond",
+    namespace=None,
```

Comment:
As in the yaml comment, I also think this adds an extra line for new users to understand without really being needed.

Reply:
This is the default behaviour from hera right now, happy to change it.